requirements-devel.txt

```diff
@@ -120,4 +120,4 @@ python-apt @ https://launchpad.net/ubuntu/+archive/primary/+sourcefiles/python-a
 PyNaCl==1.4.0; sys.platform != "linux"
 PyNaCl @ https://files.pythonhosted.org/packages/61/ab/2ac6dea8489fa713e2b4c6c5b549cc962dd4a842b5998d9e80cf8440b7cd/PyNaCl-1.3.0.tar.gz; sys.platform == "linux"
 setuptools==49.6.0
-pyinstaller==4.3; sys.platform == "win32"
+pyinstaller==4.10; sys.platform == "win32"
```

Comment:
This old version of `pyinstaller` was causing the [appveyor test](https://ci.appveyor.com/project/sergiusens/snapcraft/builds/44697101) to fail.  The [changelog](https://pyinstaller.org/en/v5.3/CHANGES.html#id54) says improvements were made in 4.4 to metadata collection.  I think that was the root cause.

Anyways, I bumped it to the latest 4.x version.  `pyinstaller` 5.x will require some tweaks to appveyor.yml.